search/Makefile

```diff
@@ -49,3 +49,7 @@ push-image:
 .PHONY: build-push-image
 build-push-image-latest: image push-image
 build-push-image-version: image-version push-image-version
+
+.PHONY: install_deps
+install_deps:
+	pip3 install -e ".[all]"
```

Comment:
for each of the `install_deps` in the make files, is the `-e` strictly required when doing pip install? 

I'm trying to think if there is a developer experience where the dev may do `pip install .[all]` and if it could fail.

Reply:
it will fail if you skip `-e`. Without `-e` it makes a copy of a source dir to /tmp and conducts installation there. When this copy operation is conducted symlinks are not resolved so requirements-common.txt (or requirements-dev.txt) are not copied too. Adding -e flag changes the process in such way that files are not copied to /tmp beforehand but installation is conducted in source dir.

Reply:
Got it. Trying to think through a scenario where someone makes changes / installs from source but forgets to add the `-e` flag. Is it worth updating the readme / docs somewhere to mention this?